site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -32,11 +32,21 @@ public function showQueue() {
             );
         }
 
+        $queries = array();
+        parse_str($_SERVER['QUERY_STRING'], $queries);
+
+        $full_history = false;
+        if (array_key_exists('full_history', $queries) && $queries['full_history'] === "true") {
+            $full_history = true;
+        }
+
+        $this->core->getOutput()->addInternalCss('officeHoursQueue.css');
```

Comment:
This line for adding CSS needs to be put in `OfficeHoursQueueViewer` to conform with MVC pattern.

Reply:
I put it in the OfficeHoursQueueViewer constructor, is that the expected place I should have put it?

Reply:
Oh sorry I misunderstood the code. It is supposed to be in `OfficeHourQueueView`.

Reply:
https://github.com/Submitty/Submitty/blob/90437c056fb7fa8d6b5de2081df52d4a376d3b64/site/app/views/OfficeHoursQueueView.php#L11

here

Reply:
Wait, I think you may have meant to say it should be put in `OfficeHoursQueueView`, am I correct?

`OfficeHoursQueueView` is the view for the office hour queue.
`OfficeHoursQueueViewer` is a model that stores some information for the person viewing the office hour queue, making them a "viewer".

Maybe it would make sense for me to rename `OfficeHoursQueueViewer` to `OfficeHoursQueueModel` or `OfficeHoursQueueUser`?